fastlane/Fastfile

```diff
@@ -0,0 +1,43 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+opt_out_usage
+
+default_platform(:ios)
+
+pods = [
+  "Amplify.podspec",
+  "AmplifyPlugins.podspec",
+  "AmplifyTestCommon.podspec",
+  "AWSPluginsCore.podspec",
+  "CoreMLPredictionsPlugin.podspec"
+]
+
+platform :ios do
+  desc "Bump pod versions"
+  lane :bump_podspecs_patch do
+
+    pods.each { |pod| version_bump_podspec(path: pod, bump_type: "patch") }
```

Comment:
Does this bump the version for pod? Pod version and our github code base version should match

Reply:
Yes but that action is not actually utilized at the moment. I just wrote it with the intention of using it down the road. 

The plan is to add Git tagging and automatic semver versioning with conventional commits to the pipeline.